api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/NewCardScheduleService.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.dhsc.htbhf.claimant.scheduler;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.repository.ClaimRepository;
+
+import java.util.stream.Stream;
+
+@Service
+@AllArgsConstructor
+@Slf4j
+public class NewCardScheduleService {
+
+    private ClaimRepository claimRepository;
+
+    /**
+     * Print out first 10 new claims (testing purposes only).
+     */
+    @Transactional
+    public void createNewCards() {
+        Stream<Claim> newClaims = claimRepository.getNewClaims();
```

Comment:
I think it might be worth moving the call to `claimRepository.getNewClaims()`  and subsequent forEach into the CreateCardJob, and rename NewCardScheduleService to something like `NewCardRequestService` - whose method is transactional but operates on a single claim.

Reply:
We could rename it to NewCardService, it doesn't need to know that it's a scheduled service.
Why would the method have to be transactional? If the job is getting the data then this service won't be doing any database operations.

Reply:
Moved iterating over new claims to job.

Reply:
Moved iterating over new claims to job.